js/tabs.js

```diff
@@ -481,3 +482,11 @@ Tabs.prototype.onDocChanged_ = function(e, session) {
 
   tab.changed();
 };
+
+/**
+ * Determines whether any tabs are open.
+ * @return {boolean} True if at least one tab is open.
+ */
+Tabs.prototype.hasOpenTab = function() {
+  return Boolean(this.tabs_.length);
```

Comment:
Just do

!!this.tabs_.length

to coerce it.

Reply:
Done